geneocr-ui/1.0/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# base image
+FROM node:13.12.0-alpine as build
+
+# metadata
+LABEL base_image="node:13.12.0-alpine"
```

Comment:
LABEL elts must be set in the "final" image, not the build image
In your case, after FROM nginxinc/nginx-unprivileged:stable-alpine line 43
else final image won't be tagged with labels

Reply:
Thanks for the feedback, will make the necessary changes